sdk/appservice/arm-appservice-rest/README.md

```diff
@@ -0,0 +1,59 @@
+# Azure WebSiteManagement client library for JavaScript
+
+App Service Client
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/appservice/arm-appservice-rest)
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/arm-appservice)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/arm-appservice)
+- [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-an-authentication-token) to use this package.
+
+### Install the `@azure-rest/arm-appservice` package
+
+Install the Azure WebSiteManagement client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/arm-appservice
+```
+
+### Create and authenticate a `WebSiteManagementClient`
+
+To use an [Azure Active Directory (AAD) token credential](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-an-authentication-token),
+provide an instance of the desired credential type obtained from the
+[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
+
+To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) 
+
+After setup, you can choose which type of [credential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) from `@azure/identity` to use.
+As an example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential)
+can be used to authenticate the client:
```

Comment:
Do we miss the sample code here?

Reply:
As we don't have the documentation for RLC, is it possible to include some samples in readme file?

Reply:
Updated with the code example here https://github.com/Azure/azure-sdk-for-js/pull/22044/files#diff-d87941b67a477905ad36e74962330a2ce64f0a2d2904f66f299a25ceb4634c4dR56